guides/building_an_asset_processing_framework.md

```diff
@@ -2,6 +2,29 @@
 
 This guide is for using a Sprockets::Environment to process assets. You would use this class directly if you were building a feature similar to Rail's asset pipeline. If you aren't building an asset processing frameworks, you will want to refer to the [End User Asset Generation](end_user_asset_generation.md) guide instead. For a reference use of `Sprockets::Environemnt` see [sprockets-rails](github.com/rails/sprockets-rails).
 
+## Gzip
+
+By default when sprockets generates a compiled asset file it will also produce a gzipped copy of that file. Sprockets only gzips non-binary files such as CSS, javascript, and SVG files.
```

Comment:
There are some instances of "sprockets" with lower-case "s", does that refer to the executable *bin/sprockets*?